added two new entities: creditcard and bank tables. issue: #77

zaynba opened the pull request:
- added two new tables: credit card and bank. these two tables reference the paymentID in payment table through cardID and bankID 
- also updated the two tables ticketinventory and ticketinfo to both have ticketID 
- also updated payment table to have payoutID that references the sellerID inside the seller table 
- created dummy data for the two new tables: credit card and bank. 
- updated the events table to include the status which has data types of enum values: "approved", "pending", "rejected". 
- additionally, inside the events table I have also added adminID that references the adminID inside the admin table.
this is issue: #77

tatkg24 committed:
Merge pull request #63 from JoheeYeom0103/main

Merge to Main

zaynba committed:
Update .DS_Store

JoheeYeom0103 committed:
Merge pull request #75 from JoheeYeom0103/main

Merge new changes into my branch

JoheeYeom0103 committed:
Update Paymentinfo & Payoutinfo Pages

Added php script to setup the environment to bring data from database. However, the ddl does not include the data the pages need.

JoheeYeom0103 committed:
Merge branch 'Johee' of https://github.com/JoheeYeom0103/TicktHub into Johee

zaynba committed:
added two new entities: creditcard and bank tables.

zaynba committed:
trying to pull, but i need to merge DS store?

kevinruann committed:
Add files via upload

kevinruann committed:
Rename Week6.docx to W6

kevinruann committed:
Rename W6 to W6.docx

kevinruann committed:
Rename Week7.pdf to W7.docx

kevinruann committed:
Rename W7.docx to W7.pdf

tatkg24 committed:
Changed to php files, and updated workspace.

tatkg24 committed:
Changed pages from html to php for local hosting purposes. Completed backend functionality for adminDashboard. Created an admin profile page. Tested the new implementation by conducting user heuristic testing and summarizing the results.

kevinruann reviewed (approved):
DDL looks great. The dummy data is a great addition for testing, the table information is pretty good so far, it might have to change slightly once we develop more of the backend. Awesome work!

JoheeYeom0103 committed:
Update SQL With Sample Data

// Changes Made // 
1) Each user now can be only buyer, seller, or administrator 
2) Payment now includes every credit card and bank account information
3) Credit Card and Bank Transfer entities now have userId as well
4) Payment information including Credit Card and Bank Transfer now all have the same holder name
5) Each user now has multiple options for credit card and bank account stored

JoheeYeom0103 committed:
Add Payment & Payout info Backend

1) Connected to database
2) Initially display data from the database
3) Dynamically delete the previously saved payment informations for credit card and bank transfer
4) Dynamically add a new payment information for credit card and bank transfer
5) Server side validation in PHP and visual indication

zaynba committed:
fixing conflicts ??

JoheeYeom0103 committed:
Merge pull request #84 from JoheeYeom0103/Johee_Backend

Update SQL To Resolve Discrepency

zaynba committed:
completed php for signupAction. loginAction, logoutAction, and i have changed the user table in the ddl

kevinruann committed:
Merge pull request #81 from JoheeYeom0103/Kevin

Added backend functionality to browse tickets and back button

tatkg24 committed:
Completed Backend for Buyer - View Purchased Tickets and Seller Manage Events/Sales Info

tatkg24 committed:
Merge branch 'main' of https://github.com/JoheeYeom0103/TicktHub into Tatum

tatkg24 committed:
fixed conflicts

zaynba committed:
changed ddl to match joy's ddl. I've also changed user table

zaynba committed:
changed user's phone numbers to be different

zaynba committed:
fixed syntax error in ddl

zaynba committed:
fixed duplicate email error for users

tatkg24 committed:
Created unit tests for previous code, refactored some pages/methods to fix DB connect issue, created .gitignore file

tatkg24 committed:
Combined Code for M4

tatkg24 committed:
New SQL for combined site

tatkg24 committed:
Added Kevin's updated browse tickets pages and included the composer.json file

kevinruann committed:
Add files via upload

JoheeYeom0103 committed:
Create laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

tatkg24 committed:
Vendor files and composer lock file

tatkg24 committed:
Merge branch 'm4Submission' of https://github.com/JoheeYeom0103/TicktHub into m4Submission

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

tatkg24 committed:
payout script url change

tatkg24 committed:
Merge branch 'm4Submission' of https://github.com/JoheeYeom0103/TicktHub into m4Submission

JoheeYeom0103 committed:
Delete tests/TransactionTest.php

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

tatkg24 committed:
Chnaged phpunitxml file and removed from gitignore

tatkg24 committed:
Merge branch 'm4Submission' of https://github.com/JoheeYeom0103/TicktHub into m4Submission

JoheeYeom0103 committed:
Update phpunit.xml

Remove <php>

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

tatkg24 reviewed (approved):
Thanks for adding these Zaynb! This will help us in testing and building out the DB functionality of our code!

tatkg24 committed:
updated some links, added more user heuristic tests, created todos for payment/payout bugs, fixed signup to account for buyers/sellers

tatkg24 committed:
Merge branch 'm4Submission' of https://github.com/JoheeYeom0103/TicktHub into m4Submission

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update laravel.yml

JoheeYeom0103 committed:
Update AdminTest.php

Updated DB connection info to configure for Github Action Workflow

DB_USER and DB_PASSWD -> DB_USERNAME and DB_PASSWORD

JoheeYeom0103 committed:
Update ManageEventsTest.php

Updated DB Connection to configure for Github Action workflow

JoheeYeom0103 committed:
Update AdminTest.php

Update the database connection to be consistently configured

JoheeYeom0103 committed:
Update ManageEventsTest.php

Update database connection to be consistently configured

JoheeYeom0103 committed:
Update laravel.yml

Update php setup

JoheeYeom0103 committed:
Update laravel.yml

zaynba committed:
Merge branch 'm4Submission' of https://github.com/JoheeYeom0103/TicktHub into Zaynb

JoheeYeom0103 commented:
Great job including paymentId in the two different payment method tables to enable transaction! If you have made any new changes other than it would be you can reference this issue in your new issue!

zaynba committed:
loginTest.php for logging in needs to be tested. (under testing folder), signupTest still needs to be created

zaynba committed:
created signupTest.php, have not tested it -- im pretty sure its wrong :(

zaynba committed:
trying to fix error, commented out database include
